source/_patterns/20-molecules/image-carousel/image-carousel.scss

```diff
@@ -156,9 +156,18 @@
   &:active {
     background-color: $color-srf-neutral-blacka80;
   }
+}
+
+.has-touch {
```

Comment:
Halt Stopp, BEM-Polizei

Reply:
Oh, und warum etwas setzen und dann in einem Spezialfall überschreiben wenn man etwas nur dann setzen kann, wenn der Spezialfall nicht eintritt?

z.B. statt im hasTouch-Fall die Hintergrundfarbe auf transparent zu setzen, könntest du auch den Hover-Status nur im nicht-hasTouch-Fall überhaupt erst setzen.

Reply:
Wenn ich den Hover-Status im nicht-hasTocuh-Fall setze überschreibt der Hover-Status den Active-Status, spricht der Active-Status funktioniert nicht mehr. Wollte mir das !important sparen.

Reply:
Hä?

Deine Lösung:
```
.carousel__link--prev,
.carousel__link--next {
  &:hover { background-color: $hoverColor; }
  &:active { background-color: $activeColor; }
}

.has-touch {
  .carousel__link--prev,
  .carousel__link--next {
    &:hover { background-color: transparent; }
    &:active { background-color: $activeColor; }
  }
}
```

Meine Idee:
```
.carousel__link--prev,
.carousel__link--next {
  &:active { background-color: $activeColor; }
}

.carousel--no-touch {
  .carousel__link--prev,
  .carousel__link--next {
    &:hover { background-color: $hoverColor; }
  }
}
```

Wo ist das Problem?

Reply:
Das Problem ist, dass meine geistigen Kapazitäten Freitags offenbar limitiert sind. *facepalm*

Reply:
Also keine Garantie, gell :) Hab's nur im Kopf zusammengestiefelt, ist nicht getestet oder so 🤞

Reply:
Doch doch habs schon getestet. Hatte davor auch schon eine fast identische Lösung, nur ist mir da ein kleiner, dummer Fehler unterlaufen, der das oben erwähnte Problem verursachte.

Reply:
Dann ab ins Testing damit